src/dom/dom.js

```diff
@@ -2671,169 +2983,124 @@ class MediaElement extends p5.Element {
+   *   if (isPaused === true) {
+   *     text('Double-click to start!', 50, 50);
+   *   } else {
+   *     text('Double-click to pause!', 50, 50);
+   *   }
    * }
    *
-   * function mouseClicked() {
-   *   //here we test if the mouse is over the
-   *   //canvas element when it's clicked
-   *   if (mouseX >= 0 && mouseX <= width && mouseY >= 0 && mouseY <= height) {
-   *     background(200);
-   *
-   *     if (sampleIsPlaying) {
-   *       //Calling pause() on our
-   *       //p5.MediaElement will stop it
-   *       //playing, but when we call the
-   *       //loop() or play() functions
-   *       //the sample will start from
-   *       //where we paused it.
-   *       ele.pause();
-   *
-   *       sampleIsPlaying = false;
-   *       text('Click to resume!', width / 2, height / 2);
-   *     } else {
-   *       //loop our sound element until we
-   *       //call ele.pause() on it.
-   *       ele.loop();
-   *
-   *       sampleIsPlaying = true;
-   *       text('Click to pause!', width / 2, height / 2);
-   *     }
+   * function doubleClicked() {
+   *   if (isPaused === true) {
+   *     // If the beat is paused,
+   *     // play it.
+   *     beat.play();
+   *     isPaused = false;
+   *   } else {
+   *     // If the beat is playing,
+   *     // pause it.
+   *     beat.pause();
+   *     isPaused = true;
    *   }
    * }
-   * </code></div>
+   * </code>
+   * </div>
    */
   pause() {
     this.elt.pause();
     return this;
   }
 
   /**
-   * Set 'loop' to true for an HTML5 media element, and starts playing.
+   * Play the audio/video repeatedly in a loop.
    *
    * @method loop
    * @chainable
    * @example
-   * <div><code>
-   * //Clicking the canvas will loop
-   * //the audio sample until the user
-   * //clicks again to stop it
-   *
-   * //We will store the p5.MediaElement
-   * //object in here
-   * let ele;
-   *
-   * //while our audio is playing,
-   * //this will be set to true
-   * let sampleIsLooping = false;
+   * <div>
+   * <code>
+   * let beat;
    *
    * function setup() {
-   *   //Here we create a p5.MediaElement object
-   *   //using the createAudio() function.
-   *   ele = createAudio('assets/lucky_dragons.mp3');
    *   background(200);
-   *   textAlign(CENTER);
-   *   text('Click to loop!', width / 2, height / 2);
-   * }
    *
-   * function mouseClicked() {
-   *   //here we test if the mouse is over the
-   *   //canvas element when it's clicked
-   *   if (mouseX >= 0 && mouseX <= width && mouseY >= 0 && mouseY <= height) {
-   *     background(200);
+   *   textAlign(CENTER);
+   *   text('Double-click to stop!', 50, 50);
    *
-   *     if (!sampleIsLooping) {
-   *       //loop our sound element until we
-   *       //call ele.stop() on it.
-   *       ele.loop();
+   *   // Create a p5.MediaElement using createAudio().
+   *   beat = createAudio('assets/beat.mp3');
+   *   // Play the beat in a loop.
+   *   beat.loop();
```

Comment:
I think this one might not work when navigating directly to the page since it's trying to autoplay sound. Maybe this could be a muted video so you can see it play immediately? Or maybe double clicking starts the loop instead of stops it?

Reply:
I haven't tried in awhile but last I remember double click can be a bit flaky on touch screens (they will zoom instead) so might want to consider just using single click overall.